CMakeLists.txt

```diff
@@ -103,6 +105,25 @@ foreach(varName HAS_STD_LOCALE  HAS_ODBC  HAS_MBSRTOWCS  HAS_WCSTOMBS  HAS_FWIDE
   endif()
 endforeach()
 
+#
+# Package and sign if Apache maintainer
+#
+option(APACHE_MAINTAINER "Apache maintainer" OFF)
+if(APACHE_MAINTAINER)
+set(CPACK_SOURCE_PACKAGE_FILE_NAME "apache-log4cxx-${LOG4CXX_RELEASE_VERSION}")
+set(CPACK_SOURCE_GENERATOR "TGZ;ZIP")
+set(CPACK_SOURCE_IGNORE_FILES ".git/;build/")
+include(CPack)
+
+add_custom_target( dist 
```

Comment:
How do you execute that target, on the shell and with which command? Visual Studio doesn't seem to provide anything obvious in its UI, `installing` doesn't seem to trigger that target. Adding `ALL` like for Doxygen does, but that's not what you intended?

Reply:
At least on *nix, this creates a 'make dist' target.  With Visual Studio, you need to switch to 'CMake Targets View' in the Solution Explorer.  You can then run the target from there.

Reply:
thanks, found it with a little help:

https://devblogs.microsoft.com/cppblog/cmake-support-in-visual-studio-targets-view-single-file-compilation-and-cache-generation-settings/

![Clipboard01](https://user-images.githubusercontent.com/6223655/104340677-3c49ee00-54f9-11eb-819c-37108c4b6e09.png)
![Clipboard02](https://user-images.githubusercontent.com/6223655/104340680-3ce28480-54f9-11eb-9693-af9b9c523dc9.png)
![Clipboard03](https://user-images.githubusercontent.com/6223655/104340682-3d7b1b00-54f9-11eb-8a23-945f4a7eeec7.png)